zephyr/Kconfig

```diff
@@ -29,13 +29,11 @@ config SOF_ZEPHYR_STRICT_HEADERS
 	  If unsure, say n.
 
 config DMA_DOMAIN
-	bool "Experimental: Enable the usage of DMA domain."
+	bool "Enable the usage of DMA domain."
+	default y if IMX
 	default n
```

Comment:
nitpick: is `default n` really required? It should be set to `n` if CONFIG_IMX is not enabled. Otherwise, LGTM.

Reply:
Yes, I think you are right. It is set to `n`. Will update.